libraries/AP_HAL_ChibiOS/hwdef/Pixhawk6C/defaults.parm

```diff
@@ -0,0 +1,13 @@
+# setup correct defaults for battery monitoring for holybro power brick
+BATT_MONITOR 4
+BATT_AMP_PERVLT 36.36
+BATT_VOLT_MULT 18.18
+
+BATT2_AMP_PERVLT 36.36
+BATT2_VOLT_PIN 18.18
```

Comment:
Looks like these are both covered by `HAL_BATT_VOLT_SCALE` and `HAL_BATT_CURR_SCALE`